src/clj/fluree/db/query/fql/syntax.cljc

```diff
@@ -140,7 +140,15 @@
                          [:file-data? {:optional true} :boolean]
                          [:index-files-ch {:optional true} :any]
                          [:time {:optional true} :string]]
-    ::txn-opts          [:and ::stage-opts ::commit-opts]
+    ::ledger-opts       [:map
```

Comment:
I don't think these options are valid for any transaction, but only when a new ledger is created. I think it would be better for there to be separate create and transact option schemas.